src/sass/partials/_footer.scss

```diff
@@ -6,16 +6,22 @@
   color: $color-main-background;
   font-family: $Font-secondary;
   padding-left: 20px;
+  padding-top: 50px;
+  width: 472px;
```

Comment:
nie moze miec takiej szerokosci, bo strona my byc responsywna a nie bedzie przy takiej szerokosci, bo to bedzie blokowalo

Reply:
albo ponizej 320px, albo width 100%